src/features/commentsSlice.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/no-cycle */
```

Comment:
> /* eslint-disable import/no-cycle */

You should avoid using cycle imports, please try to fix this

> /* eslint-disable no-param-reassign */

If you don't need a specific ESLint rule, it's better to disable it directly in the config. (you repeated `/* eslint-disable no-param-reassign */` 6 times)